### DISCUSSION TITLE
[HELP!!][taiko+gauge] test exit with error Timeout when click some buttion

### DISCUSSION THREAD

**Discussion by hw64220** [getgauge/taiko#2690] [General]
1.screenshot.png is the error screenshot
2.question.png is the code sample, I have highlight the code when it occurs error. below is the error message
3.error.gif is the process the error happens

please give  me some advice why my test case failed by the timeout error so frequently? Is this caused by switching between popup window?

<img width="690" alt="screenshot-801622069237850" src="https://user-images.githubusercontent.com/7598828/220868893-14685680-278e-4f6b-8dcb-fe2d5d771e71.png">

![question1](https://user-images.githubusercontent.com/7598828/220868168-30a28f1e-dca8-4409-ad34-2a2bebda9dbd.png)

![error](https://user-images.githubusercontent.com/7598828/220868209-3e3f781d-aa42-4f9f-a085-0340566044fb.gif)

**Comment by hw64220** [comment]
can anyone help on this?

**Comment by NivedhaSenthil** [comment]
This timeout is happening from Gauge, seems the step is taking more time. You can either split the step or increase the step timeout in the properties like mentioned [here](https://docs.gauge.org/configuration.html?os=macos&language=javascript&ide=vscode).

**Reply by hw64220** [reply]
thx Nivedha, I will have a try.
